### ISSUE TITLE
Issues to reconnecting the Reader-iOS SDK

### ISSUE THREAD

**Issue by jadhavarun** [CardFlight/cardflight-ios#73]
Hello Team,

I am having an issue with transaction  Basically, I'd written down the steps what I'm doing:
1.Creating an object in viewDidLoad.
2.Creating credentials with a custom method and calling it from didReceiveCardReaderEvent only when the reader is connected.
3.Then payment is being done as expected with only single payment.
4.And go to second view controller and coming back to view controller the code has been made.
5.Now the card is not been detected.

basically, it seems the issue is with multiple payments.

Please let me know if somebody has gone through the same and succeeded in doing same.

Thanks in anticipation.
